src/Dodo.HttpClient.ResiliencePolicies/RetryPolicySettings/JitterRetryPolicySettings.cs

```diff
@@ -21,9 +21,22 @@ public class JitterRetryPolicySettings : IRetryPolicySettings
 		}
 
 		public JitterRetryPolicySettings()
+		: this(Defaults.Retry.RetryCount)
 		{
-			RetryCount = Defaults.Retry.RetryCount;
-			SleepDurationProvider = _defaultSleepDurationProvider(RetryCount, _defaultMedianFirstRetryDelay);
+		}
+
+		public JitterRetryPolicySettings(int retryCount)
+		: this(retryCount,
+			TimeSpan.FromMilliseconds(Defaults.Retry.MedianFirstRetryDelayInMilliseconds))
+		{
+		}
+
+		public JitterRetryPolicySettings(
+			int retryCount,
+			TimeSpan medianFirstRetryDelay)
```

Comment:
Maybe we should remove MedianFirstRetryDelay property? It looks strange to provide it in constructor and also we have the property with public setter.